packages/x-charts-pro/src/FunnelChart/seriesConfig/seriesProcessor.ts

```diff
@@ -27,6 +27,13 @@ const seriesProcessor: SeriesProcessor<'funnel'> = (params) => {
   seriesOrder.forEach((seriesId) => {
     const currentSeries = series[seriesId];
 
+    const firstDataPoint = currentSeries.data.at(0);
+    const lastDataPoint = currentSeries.data.at(-1);
+    const dataDirection =
+      firstDataPoint && lastDataPoint && firstDataPoint.value < lastDataPoint.value
```

Comment:
If `firstDataPoint` or `lastDataPoint` are zero, can't this be problematic? 

Probably not, I'm not seeing a use case where a pyramid/funnel chart has a zero value, but just wanted to make sure.